tests/unit/neptune/new/client/abstract_tables_test.py

```diff
@@ -217,22 +217,7 @@ def test_get_table_as_table_entries(
             progress_bar=None,
         )
 
-    @patch.object(NeptuneBackendMock, "search_leaderboard_entries")
-    def test_table_limit(self, search_leaderboard_entries):
-        # given
-        now = datetime.now()
-        attributes = self.build_attributes_leaderboard(now)
-
-        # and
-        empty_entry = LeaderboardEntry(str(uuid.uuid4()), [])
-        filled_entry = LeaderboardEntry(str(uuid.uuid4()), attributes)
-        search_leaderboard_entries.return_value = [empty_entry, filled_entry]
-
-        # and
-        for limit, expected_len in [(1, 1), (2, 2), (3, 2), (10_000, 2)]:
```

Comment:
Those tests no longer make sense if we don't perform limiting in the `HostedNeptuneBackend` but in the `searching_entries` module